portality/templates/api/v2/api_docs.html

```diff
@@ -8,8 +12,6 @@
 
 {% endblock %}
 
-{% extends "layouts/base.html" %}
```

Comment:
This breaks the swagger UI stuff, so it's not working on this branch. I agree it should definitely be at the top though.

Reply:
Hmm  I tried putting it back but  the API content still isn’t being displayed...

Reply:
I'll take this one on